docs/install.rst

```diff
@@ -1,115 +1,146 @@
+        Double click the executable file downloaded from
+        the links above.
 
-    conda install --channel conda-forge --channel defaults scipy matplotlib
+        Once the installer has completed you should have a new "miniforge
+        Prompt" entry in your start menu.
 
-or::
+In a new terminal (miniforge Prompt on Windows) run ``conda list`` to test that the install has worked.
 
-    conda install --channel conda-forge --channel defaults scipy matplotlib \
-      h5py beautifulsoup4 html5lib bleach pandas sortedcontainers \
-      pytz setuptools mpmath bottleneck jplephem asdf-astropy pyarrow
+Create Python Environment
+=========================
 
-To also be able to run tests (see below) and support :ref:`builddocs` use the
-following. We use ``pip`` for these packages to ensure getting the latest
-releases which are compatible with the latest ``pytest`` and ``sphinx`` releases::
+To create a new Python environment for ``astropy`` and other packages, start by
+launching a terminal (under a UNIX-like system) or the miniforge Prompt (under Windows).
+Now we will create and activate a new virtual environment to install ``astropy`` into:
 
-    python -m pip install pytest-astropy sphinx-astropy
+.. code-block:: bash
 
-.. warning::
+    $ conda create --name astropy python
+    $ conda activate astropy
 
-    Attempting to use `pip <https://pip.pypa.io>`__ to upgrade your installation
-    of ``astropy`` itself may result in a corrupted installation.
+In this case the environment we have created is named ``astropy`` but you can use any
+name you like.
 
-.. _testing_installed_astropy:
+In the future when you make a new terminal, you will need to run ``conda activate
+astropy`` to activate this environment.
 
-Testing an Installed ``astropy``
-================================
+.. _installing-astropy-with-pip:
 
-See the `documentation on how to test your installed version of
-astropy <https://docs.astropy.org/en/latest/development/testguide.html#running-tests-installed-astropy>`_.
+Install ``astropy``
+===================
+
+To install ``astropy`` and your choice of :ref:`dependencies <astropy-main-req>`, run
+one of the following commands::
+
+    python -m pip install astropy                # Minimum required dependencies
+    python -m pip install "astropy[recommended]" # Recommended dependencies
+    python -m pip install "astropy[all]"         # All optional dependencies
+    python -m pip install "astropy[dev_all]"     # All optional and test dependencies
+
+In most cases, this will install a pre-compiled version of ``astropy`` (called a
+*wheel*). However, if you are installing astropy on an uncommon platform, astropy will be
+installed from a source file. In this unusual case you will need a C compiler to be
+installed (see `Build from source`_ below) for the installation to succeed.
+
+.. warning:: Do **not** install ``astropy`` or other packages using ``sudo``.
```

Comment:
Might want to clarify a bit. `sudo` does not make sense for Windows user. Maybe should use more general term like "do not install using elevated privilege" or something like that. And say that it is general security practice.

Reply:
Fixed.